tools/node_modules/eslint/node_modules/eslint-plugin-jsdoc/index.js

```diff
@@ -0,0 +1,7 @@
+/**
+ * Obtained originally from {@link https://github.com/eslint/eslint/blob/master/lib/util/source-code.js#L313}
```

Comment:
This link doesn't seem to point to anything

Reply:
This would need to be fixed upstream.